x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/EnrollNodeToCluster.java

```diff
@@ -84,6 +87,10 @@
     private final OptionSpec<String> enrollmentTokenParam = parser.accepts("enrollment-token", "The enrollment token to use")
         .withRequiredArg()
         .required();
+    private final OptionSpec<Void> force = parser.acceptsAll(
```

Comment:
The `force` option means we overwrite the existing `elasticsearch.yml` but because the node might already be part of a cluster, it's not guaranteed that it will actually allow the node to enroll to the new cluster. And if it doesn't, the admin is then in a state with mismatching yml and cluster configurations.

Ultimately it is a product decision what such an option should do, but it should be clearly spec-ed and well thought out. As it currently stands, I'm not a fan. Any exiting node joining a new cluster brings along its indices, and at that point I'm not sure this can still be called "enrollment".

For reference, there is a `DetachClusterCommand` cmd line that is very verbose about the consequences of detaching the node from an existing cluster.

Reply:
> Ultimately it is a product decision what such an option should do, but it should be clearly spec-ed and well thought out.

Let's focus on that then. What are your thoughts? This aims to cover a very concrete use case, which is additional nodes in packaged installations that got auto-configured to be "first-nodes" by default during package installation. We discussed this between me you and Fabio on slack on July 16th. 

Would a warning and an additional [y/N] prompt for verification alleviate your concerns about this being used mistakenly for other purposes?